src/Hl7.Fhir.Support/Utility/Unit.cs

```diff
@@ -0,0 +1,18 @@
+﻿/* 
+ * Copyright (c) 2020, Firely (info@fire.ly) and contributors
+ * See the file CONTRIBUTORS for details.
+ * 
+ * This file is licensed under the BSD 3-Clause license
+ * available at https://raw.githubusercontent.com/FirelyTeam/fhir-net-api/master/LICENSE
+ */
+
+namespace Hl7.Fhir.Utility
+{
+    public struct Unit
```

Comment:
What is this? Looks very generic

Reply:
Yep.  Necessary for Result<T>.

Reply:
Would be familiar to people using functional languages and Result<T>.  But it is not too visible if you don't know about it.

Reply:
I'll add a bit of comment